app/views/spaces_new.erb

```diff
@@ -1,31 +1,31 @@
 <% @title="List a Space | MakersBnB" %>
 
```

Comment:
If you're going to use this view for new spaces and updating spaces, perhaps the title should change as well depending on what the page is being used for?